lib/core-client/src/preview/start.ts

```diff
@@ -13,6 +13,7 @@ import { executeLoadableForChanges } from './executeLoadable';
 
 const { window: globalWindow, FEATURES } = global;
 
+// TODO: talk to tom about the removal of this
```

Comment:
@tmeasday let's chat about this, how to remove.

Reply:
We are dropping this also.